tensor2tensor/data_generators/mathematical_language_understanding.py

```diff
@@ -89,6 +88,7 @@ def generate_samples(self, data_dir, tmp_dir, dataset_split):
     Yields:
       The data examples.
     """
+
```

Comment:
This line looks irrelevant and generally in our style guide we discourage newlines just before start of a function, please revert the whole file if no other changes are there.